lib/owl_plot.ml

```diff
@@ -1593,14 +1593,24 @@ let scatterhist = None
 
 (* other plots *)
 
+let _rotate270_ x =
+  let w, h = Owl_dense_matrix.D.shape x in
```

Comment:
Thanks! Using `slice` rotate the matrix for 180 degree quite concisely. I'm looking at how a -90 degree rotation can be achieved with it and other existing matrix manipulation functions